diff --git a/lib/dumper.js b/lib/dumper.js
index 01775b346be6bfe0e3bcee4e6f2788480ac060e9..6dd3635d599b513aebc054b15d5e239d3a7c0bf0 100644
--- a/lib/dumper.js
+++ b/lib/dumper.js
@@ -842,7 +842,7 @@ function writeNode(state, level, object, block, compact, iskey, isblockseq, obje
       singleLineOnly = SINGLE_LINE_KEYS[objectKey];
       if (block && (Object.keys(state.dump).length !== 0) && !singleLineOnly) {
         var doubleLine = state.blankLines ?
-          (objectKey === 'packages' || objectKey === 'importers' || level === 0) : false;
+          (objectKey === 'packages' || objectKey === 'importers' || objectKey === 'snapshots' || level === 0) : false;
         writeBlockMapping(state, level, state.dump, compact, doubleLine);
         if (duplicate) {
           state.dump = '&ref_' + duplicateIndex + state.dump;
